elasticsearch service not loading at boot

I tried starting it manually and it failed.  Here is the status message: 

[root@ip-192-168-9-121 ec2-user]# systemctl status elasticsearch.service
● elasticsearch.service - Elasticsearch
   Loaded: loaded (/usr/lib/systemd/system/elasticsearch.service; disabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Mon 2020-03-02 23:05:24 UTC; 14s ago
     Docs: http://www.elastic.co
  Process: 3296 ExecStart=/usr/share/elasticsearch/bin/elasticsearch -p ${PID_DIR}/elasticsearch.pid --quiet (code=exited, status=1/FAILURE)
 Main PID: 3296 (code=exited, status=1/FAILURE)


Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at com.fasterxml.jackson.dataformat.yaml.YAMLParser.nextToken(YAMLParser...:372)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.xcontent.json.JsonXContentParser.nextToken(J...a:52)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.settings.Settings.fromXContent(Settings.java:645)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.settings.Settings.fromXContent(Settings.java:620)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.settings.Settings.access$400(Settings.java:82)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.settings.Settings$Builder.loadFromStream(Set...1093)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal systemd[1]: elasticsearch.service: main process exited, code=exited, status=1/FAILURE
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal systemd[1]: Failed to start Elasticsearch.
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal systemd[1]: Unit elasticsearch.service entered failed state.
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal systemd[1]: elasticsearch.service failed.


### Here is the journal output: 

[root@ip-192-168-9-121 ec2-user]# journalctl -u elasticsearch.service
-- Logs begin at Mon 2020-03-02 09:54:39 UTC, end at Mon 2020-03-02 23:05:24 UTC. --
Mar 02 23:05:22 ip-192-168-9-121.us-west-2.compute.internal systemd[1]: Starting Elasticsearch...
Mar 02 23:05:22 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in ve
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: Exception in thread "main" 2020-03-02 23:05:24,239 main ERROR No Log4j 2 configu
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: SettingsException[Failed to load settings from [elasticsearch.yml]]; nested: Jso
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at [Source: sun.nio.ch.ChannelInputStream@5e3f861; line: 94, column: 29]];
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.settings.Settings$Builder.loadFromStream(Settings.ja
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.settings.Settings$Builder.loadFromPath(Settings.java
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.node.InternalSettingsPreparer.prepareEnvironment(InternalSe
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.cli.EnvironmentAwareCommand.createEnv(EnvironmentAwareComma
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.cli.EnvironmentAwareCommand.createEnv(EnvironmentAwareComma
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:125)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.cli.Command.main(Command.java:90)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:126)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:92)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: Caused by: com.fasterxml.jackson.core.JsonParseException: Duplicate field 'clust
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at [Source: sun.nio.ch.ChannelInputStream@5e3f861; line: 94, column: 29]
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at com.fasterxml.jackson.core.json.JsonReadContext._checkDup(JsonReadContext.jav
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at com.fasterxml.jackson.core.json.JsonReadContext.setCurrentName(JsonReadContex
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at com.fasterxml.jackson.dataformat.yaml.YAMLParser.nextToken(YAMLParser.java:37
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.xcontent.json.JsonXContentParser.nextToken(JsonXCont
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.settings.Settings.fromXContent(Settings.java:645)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.settings.Settings.fromXContent(Settings.java:620)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.settings.Settings.access$400(Settings.java:82)
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal elasticsearch[3296]: at org.elasticsearch.common.settings.Settings$Builder.loadFromStream(Settings.ja
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal systemd[1]: elasticsearch.service: main process exited, code=exited, status=1/FAILURE
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal systemd[1]: Failed to start Elasticsearch.
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal systemd[1]: Unit elasticsearch.service entered failed state.
Mar 02 23:05:24 ip-192-168-9-121.us-west-2.compute.internal systemd[1]: elasticsearch.service failed.

--- a/ami/packer-template.json
+++ b/ami/packer-template.json
@@ -31,7 +31,7 @@
     {
       "type": "shell",
       "inline": [
-        "sudo mv /home/ec2-user/elasticsearch.yml /etc/elasticsearch/elasticsearch.yml",
+        "sudo mv /home/ec2-user/elasticsearch.yml /etc/elasticsearch/elasticsearch.yml && chmod -R +r /etc/elasticsearch",
         "sudo mv /home/ec2-user/init_node.sh /usr/bin/init_node.sh && chmod +x /usr/bin/init_node.sh",
         "sudo mv /home/ec2-user/init_node.service /etc/systemd/system/init_node.service && sudo chmod 644 /etc/systemd/system/init_node.service",
         "sudo systemctl enable init_node.service"

